src/formattedcode/format_json_pp.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2017 nexB Inc. and others. All rights reserved.
```

Comment:
I have have though of both json and json-pp would be in one module (eventhough there are two scan_ouput implementations)... That would avoid plugin scripts inflation. Especially since they should share a common function since the only difference in the indent arg?